package.json

```
@@ -91,7 +91,7 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^24.1.3",
     "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-orbit-internal": "^0.1.0",
+    "@kiwicom/eslint-plugin-orbit-internal": "^0.1.0",
```

You could also use to make sure there would be no version mismatch: if it finds the package locally, it will install it, regardless of the version (whereas it might look for it on NPM if it can't find the right version locally).
```suggestion
    "@kiwicom/eslint-plugin-orbit-internal": "*",
```